Paranormal/Paranormal/Panels/ToolSettings/ColorPicker/ColorPicker.xib

```diff
@@ -65,7 +69,7 @@
                         <color key="backgroundColor" name="textBackgroundColor" catalog="System" colorSpace="catalog"/>
                     </textFieldCell>
                     <connections>
-                        <binding destination="-2" name="value" keyPath="self.deg" id="Ude-MI-EwF"/>
+                        <action selector="setDegreeFromTextBox:" target="-2" id="FbA-vX-IbY"/>
```

Comment:
Cocoa bindings, while nicely binding the xib to the view controller, require state on the view controller, which (in our case) is difficult to keep synced with the state on the document. This way, the document is the official source of settings information.  When a setting is changed, the view controller updates the document directly.  This means we could add hot keys, and a change through a hot key would update the document, which would then update the toolsettings view.